packages/types/src/primitive/Generic/Call.ts

```diff
@@ -5,13 +5,15 @@
 import { FunctionArgumentMetadataLatest, FunctionMetadataLatest } from '../../interfaces/metadata';
 import { AnyU8a, ArgsDef, CallFunction, Codec, IMethod } from '../../types';
 
-import extrinsicsFromMeta from '@polkadot/api-metadata/extrinsics/fromMetadata';
+// FIXME Here we unfortunately cannot use Decorated, because that already calls
```

Comment:
This will go away if we move the `findCall` off from here and only available on the wrapper. (So in here we never decorate anything else, only the wrapper knows how that stuff works.) So happy, it is short-term.